model/clusters_mgmt/v1/network_verification_type.model

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+struct NetworkVerification {
```

Comment:
Does a struct give the `id` , `type` and `href`?

Reply:
I don't see that getting created
```
// NetworkVerification represents the values of the 'network_verification' type.
type NetworkVerification struct {
	bitmap_           uint32
	cloudProviderData *CloudProviderData
	items             []*SubnetNetworkVerification
	total             int
}
```

Where is that?

Reply:
Oh in backend, I think we just need to remove that. Let me sync with Will